.github/workflows/build-image.yml

```diff
@@ -3,7 +3,7 @@ name: build image
 on:
   push:
     branches:
-      - 'main'
+      - "main"
```

Comment:
why does main have speech marks when no other strings in the yaml do?

Reply:
🤷🏻‍♀️